core/src/edu/asu/c3simulator/screens/AllManagementScreens.java

```diff
@@ -0,0 +1,90 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Screen;
+
+import edu.asu.c3simulator.simulation.SimulationScreen;
+
+/**
+ * Enumerated Singleton to hold all sub-screens of the Management Screen. Each
+ * enumerated value represents one sub-screen.
+ * 
+ * Note: {@link #initialize(Game)} must be called prior to
+ * {@link #getInstance()}. {@link #initialize(Game)} only needs to be called
+ * once throughout the lifecycle of the program, but additional calls are
+ * harmless.
+ * 
+ * @author Moore, Zachary
+ * 
+ */
+public enum AllManagementScreens
+{
+	PRE_MARKET, CURRENT_PRODUCTS, RETIRED_PRODUCTS, CURRENT_PRODUCTS_GROWTH, PRODUCT_HISTORY, GROWTH_DEMAND, EMPLOYMENT, MARKETING, BUSINESS_DIRECTION, FUNDING, INDUSTRY, TASKS, COMPANY_PANEL, COMPLETED_TASKS, TASK_MANAGEMENT;
```

Comment:
It looks like the formatting got thrown off.

The formatter needs to be run on this